ts/src/bitget.ts

```diff
@@ -3283,9 +3283,17 @@ export default class bitget extends Exchange {
             if (method === 'publicSpotGetV2SpotMarketCandles') {
                 response = await this.publicSpotGetV2SpotMarketCandles (this.extend (request, params));
             } else if (method === 'publicSpotGetV2SpotMarketHistoryCandles') {
-                const until = this.safeInteger2 (params, 'until', 'till');
+                const until = this.safeIntegerN (params, [ 'until', 'till', 'endTime' ]);
```

Comment:
I'm not sure `endTime` will be available here because we're reading+omitting it here 
![image](https://github.com/ccxt/ccxt/assets/43336371/1e89f6a3-a223-4109-8a64-258a4faa491d)

Reply:
Okay I removed this change